netkan/netkan/cli/common.py

```diff
@@ -101,6 +102,10 @@ def ckanmeta_repo(self) -> CkanMetaRepo:
                 init_repo(self._ckanmeta_remote, '/tmp/CKAN-meta', self.deep_clone))
         return self._ckanmeta_repo
 
+    @ckanmeta_repo.setter
+    def ckanmeta_repo(self, value: CkanMetaRepo) -> None:
+        self._ckanmeta_repo = value
+
```

Comment:
I'm not sure this is required, ckanmeta_repo is a lazy loaded property and nothing external should be setting it right?

Reply:
Removed!